tcms/testplans/static/testplans/js/get.js

```diff
@@ -358,6 +368,35 @@ function toolbarEvents() {
         }
 
     });
+
+    $('#default-tester-button').click(function(ev) {
+        let selectedCases = getSelectedTestCases();
+
+        if (!selectedCases.length) {
+            alert($('#test_plan_pk').data('trans-no-testcases-selected'));
+            return;
+        }
+
+        var email_or_username = window.prompt($('#test_plan_pk').data('trans-default-tester-prompt-message'));
```

Comment:
`email_or_username` -> `emailOrUsername`

please use JS naming conventions, otherwise we are going to have to refactor all of this once we start linting the JS code

also, is there a reason to use `var` and not `let` or even `const`

Reply:
I think initially I copy paste the code from the old js, both the "prefix" and the name. I agreed about the naming, I believe the rest of the file follow the conventions. I will refactor the names and var-> let/const. Usually I would not use let/const without transpiler(babel), as soon as we introduce webpack we could drop the var keyword, but this is my opinion.

Reply:
we already use them (an other ES6 syntax) in many other places, so if something breaks, a lot of things will break.